sdk/eventhub/azure-eventhub/azure/eventhub/_configuration.py

```diff
@@ -57,7 +60,13 @@ def __init__(
         self.receive_timeout = receive_timeout
         self.send_timeout = send_timeout
         self.custom_endpoint_address = custom_endpoint_address
-        self.connection_verify = connection_verify
+        self.connection_verify: Optional[Union[str, "SSLContext"]]
+        if ssl_context:
```

Comment:
maybe we output this warning only if both ssl_context and connection_verify is specified ?